ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -15,6 +15,32 @@ resources:
     requests:
       cpu: 200m
       memory: 400Mi
+tests:
+- as: base-ubi8-python-3.8-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: base-ubi8-python-3.8
+    env:
+      IMAGE_REPO: notebooks
+      RELEASE_VERSION: main
```

Comment:
RELEASE_VERSION will be the image tag, it should be `RELEASE_VERSION: base-ubi8-python-3.8` in this case.

Therefore the mirrored image will be `quay.io/opendatahub/notebooks:base-ubi8-python-3.8`.